src/nvim/buffer_defs.h

```diff
@@ -649,7 +649,7 @@ struct file_buffer {
   bool b_p_swf;                 /* 'swapfile' */
   long b_p_smc;                 /* 'synmaxcol' */
   char_u      *b_p_syn;         /* 'syntax' */
-  long b_p_ts;                  /* 'tabstop' */
+  int b_p_ts;                   // 'tabstop'
```

Comment:
ok, should be fine then in the meanwhile  if we can accept all num options eventually becoming `int` values. (we actually have the same inconsistency for boolean options already, I will look into that case as soon as I have a bit more time).